resteasy-core/src/main/java/org/jboss/resteasy/core/MatrixParamInjector.java

```diff
@@ -7,7 +7,7 @@
 import org.jboss.resteasy.spi.ValueInjector;
 
 import javax.ws.rs.MatrixParam;
-import javax.ws.rs.NotFoundException;
+import org.jboss.resteasy.spi.NotFoundException;
```

Comment:
Is this by mistake or on purpose? AFAIR org.jboss.resteasy.spi.NotFoundException is deprecated, btw.